code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2092,3 +2092,21 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_icon_state = "planet_cracker"
 	glass_name = "Planet Cracker"
 	glass_desc = "Although historians believe the drink was originally created to commemorate the end of an important conflict in man's past, its origins have largely been forgotten and it is today seen more as a general symbol of human supremacy."
+
+/datum/reagent/consumable/ethanol/mauna_loa
+	name = "Mauna Loa"
+	description = "Extremely hot; not for the faint of heart!"
+	boozepwr = 40
+	color = "#fe8308" // 254, 131, 8
+	quality = DRINK_FANTASTIC
+	taste_description = "firey, with an aftertaste of burnt flesh"
+	glass_icon_state = "mauna_loa"
+	glass_name = "Mauna Loa"
+	glass_desc = "Lavaland, in a drink...mug...volcano...thing."
+
+/datum/reagent/consumable/ethanol/mauna_loa/on_mob_life(mob/living/carbon/M)
+	// Heats the user up while the reagent is in the body. Occasionally makes you burst into flames.
+	M.adjust_bodytemperature(10 * TEMPERATURE_DAMAGE_COEFFICIENT)
```

Comment:
Just so you know, this heats you up less than coffee does (unless you're close to or above BODYTEMP_NORMAL). You might wanna bump up the heating rate here.

Reply:
Alright, I wasn't exactly sure how much heating was enough. I took a glimpse at capsaicin oil and took a number, but I guess I severely underestimated how quickly the bonus would tally up over time. I've upped it to 25, though if that too high, I can easily adjust it down; I just don't really have much context yet for how much that it.